EAD/EAD.tex

```diff
@@ -0,0 +1,59 @@
+\documentclass[]{article}
+\usepackage{glossaries}
+\makenoidxglossaries
+
+\newglossaryentry{computer}
+{
+	name=computer,
+	description={is a programmable machine that receives input,
+		stores and manipulates data, and provides
+		output in a useful format}
+}
+
+\newglossaryentry{computer2}
+{
+	name=computer2,
+	description={is a programmable machine that receives input,
+		stores and manipulates data, and provides
+		output in a useful format2}
+}
+
+
+% Title Page
+\title{Emergent Architecture Design}
+\author{GROUP: GOAL-Integration}
+
+
+\begin{document}
+
+
+\section{Introduction}
+This group will create an integration of the Gamygdala engine in the programming language GOAL.
+\subsection{Goals}
+In this project we have multiple goals. Besides the goals between the different groups which can be found in the product vision document we also have goals that are group specific. We think that it is very important for us that we have good code quality. Also functionality is important.
+
+\subsubsection{Source code quality}
+We think that it is important to have readible code that is wel written with the correct design patterns. Other software engineering principles we have learned so far should also be used appropriately and correctly. For example: Method sizes.
+
+\subsubsection{Functionality}
+By testing in different ways we want our code to be very functional. Since our job is to create an integration, functionality is the most important goal for our team.
+
+
+\section{Software Architecture}
+When a user is creating a virtual human in the programming language GOAL, they should be able to use the Gamygdala engine in an easy and understandable way.
+\subsection{Programming languages}
+We will work in javascript and Java. This is because GOAL is written in Java and the Gamygdala engine is written in Javascript. Both these languages we have learned in the first year of the bachelor Computer Science.Mostly we will be working in the source code of GOAL and find a way to build an integration of a javascript engine in a programming language written in Java. For this we also need to have implement a port from javascript to java for Gamygdala.
```

Comment:
Het andere groepje port gamygdala naar Java, dus hoeven we geen javascript te gebruiken.